### DISCUSSION TITLE
Is Array of Multipolygon required or could it be Multipolygon (no Array?)

### DISCUSSION THREAD

**Discussion by adamjack** [ulfsri/neris-framework#13] [Q&A]
I'm no GIS expert but I see that Multipolygon is (effectively) an array of polygons, non-overlapping. Is the use of Array[Multipolygon] in the Fire Department entity required for some use cases, or perhaps not needed? (I think Multipolygon does NOT allow overlapping polygons, which I would think is ok for a district boundary, but maybe there is a nuance there I am missing.)

**Comment by JohnDavidUL** [comment]
A multipolygon collects the geometry for a given feature into one or more (non-overlapping) polygons. Any region set can include overlapping feature geometries, it's just that there can't be overlap within a single geometry. 

For most entities jurisdiction will consist of a single feature with a single polygon geometry, so a simple scalar polygon would actually suffice. There are some though for which jurisdiction has multiple features and/or geometries consisting of multiple polygons, so we typed this as an array of multipolygons.  I hope this helps!

**Comment by adamjack** [comment]
Hey John David, 

This is no big deal on my part, just a curiosity about an optimization, but it seems like we have an array of an array of polygons to hold one polygon 99% of the time, and I was wondering why a single array of polygons wouldn't work 100% of the time, since in the real world you cannot have responsibility for the same piece of land twice. Maybe this storage you've defined just makes it easier for those who work in GIS to work with the data, which I cold understand. Still, thanks for the answer and considering my questions.

regards,

Adam
